less/about/siteDetails.less

```diff
@@ -78,10 +78,16 @@ body {
     display: block;
     clear: both;
 
+    .title {
+      width: 400px;
+      display: flex;
+      justify-content: space-between;
+      margin-left: @aboutPageSectionPadding;
```

Comment:
<img width="433" alt="screen shot 2016-12-10 at 14 17 38" src="https://cloud.githubusercontent.com/assets/11330831/21071686/78d5582e-bee3-11e6-9f8b-1d6b212073c4.png">

That is the same margin-left aligned with the sortable table below